src/targets/intents/index.ejs

```diff
@@ -1,22 +1,27 @@
 <!DOCTYPE html>
 <html lang="{{.Locale}}">
-<meta charset="utf-8">
-<title><%= htmlWebpackPlugin.options.title %></title>
-<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
-<link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
-<link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
-<link rel="manifest" href="/manifest.json">
-<meta name="theme-color" content="#ffffff">
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<% _.forEach(htmlWebpackPlugin.files.css, function(file) { %>
-  <link rel="stylesheet" href="<%- file %>">
-<% }); %>
-<% _.forEach(htmlWebpackPlugin.files.js, function(file) { %>
-  <script src="<%- file %>" defer></script>
-<% }); %>
-<% if (__STACK_ASSETS__) { %>
-{{.CozyClientJS}}
-<% } %>
```

Comment:
This was actually a trick by @m4dz to let the browser insert the closing tag by itself.

Reply:
Really? What was the trick? Just keep it in the root html tag? I didn't know that behaviour and for me it's clearer with this way.

Reply:
I think the trick is to save one line of code and a few tabs.